ResponsiveWebPage/assets/scss/styles.scss

```diff
@@ -0,0 +1,724 @@
+/* From New Branch */
+.header {
+  padding: 1.25rem 2.5rem;
+  background-color: #ffffff;
+}
+
+.logo-nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 70vw;
+  margin: 0 auto;
+}
+
+.get-started__link {
```

Comment:
Use SCSS style inheritance for such types of styles on following schema: 
get-started -> &__link -> &:hover(), etc.